lib/rspec/rails/matchers/active_job.rb

```diff
@@ -134,6 +134,10 @@ def base_job_message(job)
             end
           end
 
+          def job_match?(job)
+            @job ? @job == job[:job] : true
```

Comment:
In which case @job will be nil of false?

Reply:
`@job` will be `nil` when matcher is called without explicit job, e.g.

```rb
expect {
  AJob.perform_later
}.to have_enqueued_job
```

Reply:
ok

Reply:
I just seem weird to have `job_match?(job)` that returns true when there is no job